desserts/1-medium/chocolate-cupcakes.md

```diff
@@ -0,0 +1,55 @@
+# Chocolate cupcakes
+
+This recipe is adapted from a vegetarian cookbook, and it's quickly become a favorite for its delicious frosting and fluffy base!
+
+**Prep time:** 1 hour
+
+**Serves** 12 cupcakes
+
+## Ingredients
+
+**For the cupcake**
+* 175g plain or all-purpose flour
+* 2tsp cocoa powder
+* 1.5tsp baking powder
+* 140g sugar
+* 25g ground almonds
+* 2 large eggs
+* 175g full-fat yogurt
+* 2-3 drops vanilla extract
+* 25g butter, melted
+* 2tsp rapeseed oil
+
+**For the frosting**
+* 25g dark chocolate, finely chopped
+* 1tbsp dark chocolate, grated for decorating
+* 25g butter, softened
+* 50g icing (powdered) sugar
+* 1tbsp cocoa powder
+* 1tsp semi-skimmed milk
+* 100g low-fat curd cheese
```

Comment:
What is? Cottage cheese?

Reply:
I clarified it a bit, let me know if I need to add anything more there